### ISSUE TITLE
Need Data type as SelectField with API

### ISSUE THREAD

**Issue by BeehiveSaravana** [emkay/material-ui-table-edit#33]
We can use TextField, DataPicker in 'material-ui-table-edit'. But I need SelectField in this table. Can you please guide me?

**Comment by emkay** [comment]
yeah, you would need to add `SelectField` here https://github.com/emkay/material-ui-table-edit/blob/master/index.jsx#L7 and then add a new type here https://github.com/emkay/material-ui-table-edit/blob/master/index.jsx#L89-L126 that uses the `SelectField` component and pass in all the right props.

There might be some other styling things you need to do, but that should give you a start. Would merge this in if someone wants to do it.
